### ISSUE TITLE
Add-on Title and Releases Section

### ISSUE THREAD

**Issue by Gnates** [REDxEYE/io_mesh_SourceMDL#8]
Hey, happy new year.

There are three things I wanted to mention.
First, I noticed the title of the MDL Import add-on suggests that it can import textures:
- https://i.imgur.com/2TCcTfe.png
Does this mean both the MDL and VTF add-ons will be combined into a single add-on in the future?

Second, in the future, will users be able to download 2.79/2.80 add-ons and see the changelogs from the **Releases** page:
- https://github.com/REDxEYE/io_mesh_SourceMDL/releases

Third, do you have plans to make a recognizable **icon** for the add-ons?
- Blender Icon: https://i.imgur.com/3057boe.png
- Blender Source Tools Icon: https://i.imgur.com/lSrXnlF.png
- Crowbar Icon: https://i.imgur.com/a8PQz8M.png

**Comment by REDxEYE** [comment]
Hi, thanks! Can ya add me in discord RED_EYE#8374? Issues isn't good place to discuss such things

**Comment by Gnates** [comment]
Unfortunately, I'm still struggling to sign into Discord these past couple months.
It's because I'm on a congested public network.

I looked into telegram, but it looks like the service has been compromised recently, in terms of security.
I'm on DeviantArt: https://www.deviantart.com/xcurtainx
I do have a steam account, not sure if you wish to communicate that way.
If you have an IRC channel, I should be able to connect fine.
I'm also on https://www.officialdigitalero.org as Curtain.
If you have another way to communicate, let me know.

-EDIT- I managed to get through this time. I added you on Discord.
